Documentation/csi_ug_config_create_secret.md

```diff
@@ -0,0 +1,44 @@
+# Creating a Secret
+
+Create an array secret YAML file in order to define the storage credentials \(username and password\) and address.
+
+**Important:** When your storage system password is changed, be sure to also change the passwords in the corresponding secrets, particularly when LDAP is used on the storage systems.<br />Failing to do so causes mismatched passwords across the storage systems and the secrets, causingthe user to be locked out of the storage systems.
+
+Use one of the following procedures to create and apply the secret:
+
+## Creating an array secret file
+1.  Create the secret file, similar to the following demo-secret.yaml:
+
+The `management_address` field can contain more than one address, with each value separated by a comma.
+
+    ```
+    kind: Secret
+    apiVersion: v1
+    metadata:
+      name:  demo-secret
+      namespace: default
+    type: Opaque
+    stringData:
+      management_address: demo-management-address  # Array management addresses
+      username: demo-username                      # Array username
+    data:
+      password: ZGVtby1wYXNzd29yZA==               # base64 array password
+    ```
```

Comment:
should we somehow replace this with a link to [the relevant files in this repo](https://github.com/IBM/ibm-block-csi-driver/blob/5e3006b1456efcb815e4fb43a058e5050f932b85/deploy/kubernetes/examples/demo-secret.yaml)?

Reply:
i've been trying to get the reuse content to work without any luck. i'm still looking into it from my side with others that have worked with MD / github/ and our production content. if are able to get this to work, let me know.